src/dns/aardvark.rs

```diff
@@ -249,11 +250,25 @@ impl Aardvark {
             .collect::<Vec<String>>()
             .join(",");
 
-        let data = format!(
+        let mut data = format!(
             "{} {} {} {}\n",
             entry.container_id, ipv4s, ipv6s, container_names
         );
 
+        if let Some(dns_servers) = &entry.container_dns_servers {
+            if !dns_servers.is_empty() {
+                let dns_server_collected = dns_servers
+                    .iter()
+                    .map(|g| g.to_string())
+                    .collect::<Vec<String>>()
+                    .join(",");
+                data = format!(
+                    "{} {} {} {} {}\n",
+                    entry.container_id, ipv4s, ipv6s, container_names, dns_server_collected
+                );
+            }
+        }
+
```

Comment:
why use almost the same format() string twice? just create the dns server string before and add a space at the beginning if it is not empty